invenio_base/__init__.py

```diff
@@ -24,6 +24,129 @@
 
 """Invenio application loader.
 
+Quickstart
+----------
+Invenio-Base is taking advantage of advanced patterns for building Flask
+application. It assumes you already have understanding of
+`patterns for Flask <http://flask.pocoo.org/docs/dev/patterns/>`_.
+
+Dependencies
+~~~~~~~~~~~~
+First we need to install and import dependencies.
```

Comment:
Can you end the sentence with a colon (`dependencies:`) here and everywhere else?  The ending colon is usually used to introduce a list, a definition, a quote, or an example.  

This would clearly indicate to the reader that the Python example that follows is part of this sentence, not the following one.  (In other words, that examples don't come before explanations.)  Sometimes it is  obvious, but other times it may not be, hence it is advantageous to have a unique style everywhere.

We use this style already in places like:
- http://invenio.readthedocs.io/en/latest/installation/installation-quick.html
- http://invenio.readthedocs.io/en/latest/installation/installation-detailed.html